pkg/cmd/add_repo.go

```diff
@@ -203,7 +204,8 @@ func (ad addRepoCmd) runStdin() CommandRunnerFunc {
 			return err
 		}
 
-		successMsg := fmt.Sprintf("The %q repository was added with success, now you can use your formulas with the Ritchie!", r.Name)
+		successMsg := fmt.Sprintf("The %q repository was added with success, "+
+			"now you can use your formulas with the Ritchie!", r.Name)
```

Comment:
What about identing like so
```suggestion
		successMsg := fmt.Sprintf(
		"The %q repository was added with success, now you can use your formulas with Ritchie!",
		 r.Name,
)
```

Reply:
Done